utils/bin/indexer/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "cow-amm-indexer"
+version = "0.1.0"
+edition = "2021"
+description = "Indexer for legacy^2 amms"
```

Comment:
Why is this `legacy^2`? I thought the safe based contracts are "more legacy" than the standalone ones.

Reply:
A pun in that we seem to be squaring the amount of legacy stuff out there. :laughing: